kitsune/wiki/strategies.py

```diff
@@ -296,6 +300,49 @@ def translate(self, l10n_request: TranslationRequest) -> TranslationResult:
         result.method = TranslationMethod.HYBRID
         return result
 
+    def reject_stale_translations(self) -> list[Revision]:
+        """
+        Reject stale, unreviewed machine translations.
+        """
+        if not settings.HYBRID_ENABLED_LOCALES:
+            return []
+
+        results = []
+        for rev in self.content_manager.get_stale_translations(
+            locales=settings.HYBRID_ENABLED_LOCALES
+        ):
+            self.content_manager.reject_revision(rev, comment="No longer relevant.")
+            results.append(rev)
+        return results
+
+    def publish_unreviewed_within_grace_period(self) -> list[Revision]:
```

Comment:
how about `approve_translations` | `auto_approve_translations` | `approve_pending_translations`? It's closer to the naming schema that we have so far and the details are described in the docstring